coil/src/main/java/com/google/accompanist/coil/Coil.kt

```diff
@@ -60,6 +62,30 @@ object CoilImageStateDefaults {
     }
 }
 
+@Suppress("NOTHING_TO_INLINE")
+@Composable
+inline fun rememberCoilPainter(
```

Comment:
it is not a good idea to mark such complex functions as inline. it will add a lot of code into every usage. 
so imagine the usage like this
rememberCoilPainter("url")
in this case all not specified params still need to be resolved so their default values are inlined into every usage. there is also some compose generated code around all the composables used in default values like for
CoilImageStateDefaults.defaultImageLoader()
Compose actually has to generated observation groups and so on.
inlining composable functions sometimes useful in order to save on lambda allocations, but most of the times it is not needed

Reply:
Done.